README.md

```diff
@@ -27,25 +27,25 @@ Currently, @redocly/openapi-cli supports these features:
 
 ## What makes this tool different
 
-Unlike other OpenAPI linters, `@redocly/openapi-cli` defines the possible type tree of a valid OpenAPI definition and then traverses it. This approach is very similar to how linters for programming languages work and results in major performance benefits over other approaches. Extend functionality at different points in the lifecycle with preprocessors, rules, and decorators.
+Unlike other OpenAPI linters, `@redocly/cli` defines the possible type tree of a valid OpenAPI definition and then traverses it. This approach is very similar to how linters for programming languages work and results in major performance benefits over other approaches. Extend functionality at different points in the lifecycle with preprocessors, rules, and decorators.
```

Comment:
idea: We can improve this section (not related to this PR, but didn't want to lose the idea). Something like:

- **It's faster** (uses a type tree similar to how linters for programming languages work)
- **It's highly configurable** (comes with a slew of commands and rules)
- **It's more extensible** (architected for custom plugins and types)

What do you think @skadinna ?

Reply:
I opened a PR already: https://github.com/Redocly/redocly-cli/pull/678